### PR TITLE
Changing default build distances

### PR DESCRIPTION
**Pull request by utoxin**
You asked for some suggested changes to the default reach distances. Here you go. :)

### CODE REVIEW

**Comment by gabevenberg** [comment]
seems reasonable enough! i like it.

**Comment by utoxin** [comment]
Looked through docs on Radar ranges and such, and it looks like 150 (the place range I chose is about the max normal player radar range, while the interact range is a little more than a full on radar.

I /do/ intend to look at creating the config feature you mentioned in the other issue, so for now I think this is a good range.

**Comment by gabevenberg** [comment]
according to my testing, 100 is about the reatch of the screen at full zoom out. (can almost reatch the edge to build with data.raw.player.player.build_distance = 100. 
so, maybye build distance at 50, and reatch at 125 or so?

**Comment by utoxin** [comment]
Yeah, that's probably more what I'll do for my personal play style. But I'm going to be working on another PR tonight to actually add a config interface into the in-game options, so this can be tuned easily by the player. :)

**Comment by jaguilar** [comment]
The reach and build distances should by default cover the entire zoomed out
ordinary window (equiv behavior to 0.14). Of course, I'm happy if we get
the config API implemented and people can tweak as desired without having
to edit lua files.

**Comment by utoxin** [comment]
@jaguilar And with my defaults above they DEFINITELY cover that entire area, plus a bit of extra in the radar view, without allowing interaction with extremely distant areas. :)

**Comment by jaguilar** [comment]
OK, sounds good to me. Will review and test when I have my son in bed.
Expect the changes to be merged later tonight.

**Comment by gabevenberg** [comment]
125 cover the entire view area, and a couple tiles outside.

File: long-reach_0.0.7/data-final-fixes.lua [hunk 1]
```diff
@@ -1,2 +1,5 @@
-data.raw.player.player.build_distance = 10000
-data.raw.player.player.reach_distance = 10000
+-- Rational: Pulling items out of machines is a nice timesaver, building 
+-- lets you do things like spam turrets into biter nests, so I think 
+-- building should be shorter range than adding / removing items
+data.raw.player.player.build_distance = 150
+data.raw.player.player.reach_distance = 300
```
Comment (jaguilar) [inline long-reach_0.0.7/data-final-fixes.lua]: 150 units is already a perfectly safe distance for spamming turrets in biter nests. For simplicity, please remove the rationale and set these to the same value. So long as it covers the entire viewport, I don't really care what the amount is.

**Review by jaguilar — CHANGES_REQUESTED** [review]
Please make this change, then I'll test and we can merge.

File: long-reach_0.0.7/data-final-fixes.lua (hunk 1 above)
Comment (utoxin) [reply long-reach_0.0.7/data-final-fixes.lua]: I'm about 90% done writing the version that actually has a settings GUI.

Comment (jaguilar) [reply long-reach_0.0.7/data-final-fixes.lua]: OK, sounds good to me. May not review tonight, should do tomorrow. :)

**Comment by utoxin** [comment]
Made that change for now. The config screen update has hit some snags that I'm researching, so this is the simple solution.

**Comment by utoxin** [comment]
Made obsolete by #11.

**Pull request closed without merging**